VirtPup.Test/UnitTest1.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using Xunit;
+
+namespace VirtPup.Test
+{
+    public class UnitTest1
+    {
+        [Fact]
+        public void Test1()
+        {
+
+        }
+    }
+}
```

Comment:
Borde finnas med enhetstester, speciellt eftersom det står med i Definition Of Done.

Reply:
Japp..... lol 👍🏻